### PR TITLE
Prac 5 Code Review Request

### PR DESCRIPTION
**Pull request by taitymrobinson**
Could you please review the overall formatting of code, use of naming and conciseness? @Jasper-afk

### CODE REVIEW

File: prac_05/emails.py
Comment (Jasper-afk) [inline prac_05/emails.py:1]: Does the extract_name function need to print the name?

File: prac_05/wimbledon.py
Comment (Jasper-afk) [inline prac_05/wimbledon.py:1]: In the 'process_data' function, 'winner_to_win_data' sounds a bit strange. What you already have in the 'main' and 'display' functions, 'champion_to_win_count', sounds clear to what's being conveyed. Perhaps renaming what's in the 'process_data' function to what you have everywhere else could be more consistent and easier to read?

File: prac_05/word_occurrences.py
Comment (Jasper-afk) [inline prac_05/word_occurrences.py:1]: In the for statement, 'occurrences' implies a list of values is being used when really it's just a single value. Perhaps renaming it to 'count' could show what your trying to imply?

**Comment by Jasper-afk** [comment]
Overall, the code formatting and naming are quite good, but there are a few instances of small name changes the could be made (comments added to the respective file) to improve readability. I have also noticed your docstrings do not have full stops. I to forget about the full stop sometimes. (commit messages and docstrings are just too similar)

**Comment by taitymrobinson** [comment]
Thanks for taking the time to review!

**Pull request merged**